internal/host/logscmd.go

```diff
@@ -44,3 +56,29 @@ func (i *logsCmd) GetSteps(ctx context.Context, host *models.Host) ([]*models.St
 
 	return []*models.Step{step}, nil
 }
+
+func (i *logsCmd) getMastersIPsInHostCluster(ctx context.Context, host *models.Host) ([]string, error) {
+
+	var cluster models.Cluster
+	if err := i.db.First(&cluster, "cluster_id = ?", host.ClusterID).Error; err != nil {
```

Comment:
no need for this , hosts are enough

Reply:
How will you access the `MchineConfigCidr` of the cluster without it?

Reply:
so u don't need hosts
U need one call to DB. If u bring cluster , just preload hosts

Reply:
Oh OK, I got what you meant now, you are right, I don't need hosts.